build.rs

```diff
@@ -9,7 +9,23 @@ fn main() {
     let out_dir = PathBuf::from(env::var_os("OUT_DIR").unwrap());
     let out_dir_str = out_dir.to_str().unwrap();
 
-    if cfg!(target_os = "linux") {
+    if !cfg!(target_os = "linux") {
+        panic!("Cannot use libbpf-sys on !linux");
+    }
+
+    if cfg!(feature = "novendor") {
+        let libbpf = pkg_config::Config::new()
+            .atleast_version("0.3.0")
```

Comment:
Concerned this will get out of sync with the crate version – I'll certainly forget to bump it! Can we instead do something like–

```rust
    .atleast_version(&format!("{}.{}.{}", env!("CARGO_PKG_VERSION_MAJOR"), env!("CARGO_PKG_VERSION_MINOR"), env!("CARGO_PKG_VERSION_PATCH")))
```

Since I guarantee that the major/minor/rev numbers will match the libbpf version exactly, I think this should Just Work™.